frontend/simple/views/containers/Proposals.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <dashboard-section title="Active Proposals">
+
+    <!--  Start Original Voting Banner to delete soon -->
+    <voting-banner v-for="proposal in Object.values(proposals)"
+      :proposal="proposal"
+    />
+    <!--  End Original Voting Banner to delete soon -->
+
+    <h4 class="title is-size-5 notification is-warning gi-is-banner gi-notify"
+      v-if="groupProposals.notVoted.length"
+    >
+      <i18n>These are waiting for your vote!</i18n>
+    </h4>
+    <voting
+      v-for="proposal in groupProposals.notVoted"
+      :proposal="proposal"
+      :onVoteAgainst="handleVoteAgainst"
+      :onVotedFor="handleVoteFor"
+    />
+
+
+    <h4 class="title is-size-5" v-if="groupProposals.own.length">
+      <i18n>Your own proposals</i18n>
+    </h4>
+    <voting
+      v-for="proposal in groupProposals.own"
+      :proposal="proposal"
+      :handleCloseProposal="onCloseProposal"
+    />
+
+    <i18n class="notification gi-is-banner gi-notify"
+      v-if="!showOtherProposals && allVoted"
+    >
+      Cool, you already voted on all active proposals.
+    </i18n>
+
+    <a class="gi-showOther has-text-weight-bold"
+      v-if="!showOtherProposals && groupProposals.alreadyVoted.length"
+      @click.prevent="showOtherProposals = true"
+    >
+      <i18n>Show</i18n> {{groupProposals.alreadyVoted.length}} <i18n>voted proposals</i18n>
+    </a>
+
+    <h4 class="title is-size-5" v-if="showOtherProposals">
+      <i18n>Voted proposals</i18n>
+    </h4>
+    <voting
+      v-if="showOtherProposals"
+      v-for="proposal in groupProposals.alreadyVoted"
+      :proposal="proposal"
+      :onVoteAgainst="handleVoteAgainst"
+      :onVotedFor="handleVoteFor"
+    />
+  </dashboard-section>
+</template>
+<style lang="scss" scoped>
+@import "../../assets/sass/theme/index";
+
+.gi-notify {
+  display: block;
+  margin-top: $gi-spacer-lg;
+}
+
+</style>
+<script>
+import DashboardSection from '../components/DashboardSection.vue'
+import Voting from '../components/Voting.vue'
+import VotingBanner from './VotingBanner.vue'
+// import ShowProposals from './ShowProposals.vue'
+
+export default {
+  name: 'Proposals',
+  components: {
+    DashboardSection,
+    Voting,
+    VotingBanner
+    // ShowProposals
+  },
+  props: {
+    proposals: Object
+  },
+  data () {
+    return {
+      showOtherProposals: false,
+      mockProposals: [
+        {
+          type: 'member',
+          member: {
+            picture: 'assets/images/default-avatar.png',
+            name: 'Gil'
+          },
+          title: 'Remove Member',
+          text: 'Sam had proposed to <strong>remove Gil</strong> from the group.',
```

Comment:
> Any thoughts on rewording the proposals themselves? For example, from "Karl had proposed to change..." to "Karl proposed changing...".
>
> "$user proposed $ingVerb" sounds less weird, but I'm not sure whether that would cause any troubles with l18n.

@dotmacro well, I can't have an word on this since English is not my native language :X